### ISSUE TITLE
[B17] [possibly intentional] Primal-Vis Crystal Clusters can't turn into Vitium Crystal Clusters anymore

### ISSUE THREAD

**Issue by MalkContent** [Azanor/thaumcraft-beta#1140]
As title. I realize that this might well be by choice and I don't have a preferable version of how this should work, but I thought I'd bring it up just in case this got dropped by mistake.

**Comment by Reassembly** [comment]
This seems intentional - the Thaumonomicon used to have a blurb about how Clusters could become tainted by Flux, but that paragraph has been removed. Apparently, that Flux-removal mechanic has been almost entirely replaced by Flux Condensers. (Vitium Crystals can now (technically) absorb Flux, but the necessary conditions make it a little impractical in most cases.)

Personally, I think the tainting-crystals mechanic _should_ have been kept - as a flux-management method, it had completely different costs, process, and degree of automatability than Flux Condensers. IMO, the two methods could have coexisted, being of differing usefulness in different situations.

**Comment by MalkContent** [comment]
good point, thanks. leaving this open to prevent dupes, close at will.

**Comment by Azanor** [comment]
I had to think a bit to try and remember why I changed it, then I recalled the issue I had in one test world. I was messing around with flux and this basically caused all worldspawn crystals within several chunks to switch to flux crystals which in turn caused the flux levels to drop very quickly as they spread.
I like the idea of crystals being corrupted by flux as well, but it was making flux a bit too easy to deal with and possibly screwing over people by removing their access to normal crystals. 
I will give it some more thought though. Maybe there is a middle ground somewhere.
